service-worker.js

```diff
@@ -10,6 +10,7 @@
 const GLOBAL_VERSION = 1;
 const PWA_WORKSHOP = 'pwa-workshop';
 const CACHES = {
+  PWA_WORKSHOP: `${PWA_WORKSHOP}-v${GLOBAL_VERSION + 0}`,
```

Comment:
I'm going to only use one cache. I got spoiled by Workbox in how easy it is to set up different caches with different strategies. Doing that by hand from scratch involves more work that I do not have time for. 

There will be a follow-up PR to simplify this `CACHES` object into a basic string only.